Swift/Helpers/FSExtensions/FSE+UITextView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  FSE+UITextField.swift
+//  SwiftHelpers
+//
+//  Created by Kruperfone on 31.07.15.
+//  Copyright (c) 2015 FlatStack. All rights reserved.
+//
+
+import UIKit
+
+class FSTextView :UITextView {
+    
+    private var placeholderLabel:UILabel = UILabel()
```

Comment:
I think better to do the variable as private(set) ```private(set) let placeholderLabel = UILabel()``` that we can configure font or other values in the placeholderLabel in a code another .swift of a file